_content/av/in-conversation-with-associate-professor-judson-brewer_shannon-harvey.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "In Conversation With Associate Professor Judson Brewer (Episode #06)"
```

Comment:
Don't put the Episode 6 in the title. That's what `series: ` and `number: ` are for